### ISSUE TITLE
Finish internationalization 

### ISSUE THREAD

**Issue by varmar05** [lutraconsulting/anaconda#10]
Translation into other languages seems to be only in a half-way. 

![screenshot from 2018-02-01 09-47-22](https://user-images.githubusercontent.com/23185498/35669307-f1f8f086-0734-11e8-9416-e1dda8ff69be.png)

**Comment by ICHydro** [comment]
Yes, not a priority at the moment. As long as the framework is in place, we can add the translations when relevant.
